tests/depth_limit.rs

```diff
@@ -55,5 +55,5 @@ fn depth_limit() {
         .with_new_line("\n".to_string());
     let s = ron::ser::to_string_pretty(&data, pretty);
 
-    assert_eq!(s, Ok(EXPECTED.to_string()));
+    assert_eq!(s.unwrap(), EXPECTED.to_string());
```

Comment:
why this change?

Reply:
I don't think there was any reason, and I've reverted it

Reply:
Oh I see... 

```sh
error[E0369]: binary operation `==` cannot be applied to type `std::result::Result<std::string::String, ron::ser::Error>`
  --> tests\depth_limit.rs:58:5
   |
58 |     assert_eq!(s, Ok(EXPECTED.to_string()));
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |     |
   |     std::result::Result<std::string::String, ron::ser::Error>
   |     std::result::Result<std::string::String, _>
   |
   = note: an implementation of `std::cmp::PartialEq` might be missing for `std::result::Result<std::string::String, ron::ser::Error>`
```

How is this behaving for you? It's kinda important that the tests are able to compile 😛

Reply:
yeah, we should compare error values, not strings